components/ImagesGallery/ImagesGallery.vue

```diff
@@ -808,6 +811,9 @@ export default {
           const name = response.name
           if (name) {
             this.$set(item, 'title', name.substring(0, name.lastIndexOf('.')))
+            if (name.lastIndexOf('.') === -1) {
```

Comment:
this is happening because we are removing the check for the title. I believe every image should have a title. If an image does not have one, we need to figure out why biolucida is not returning one

Reply:
This change is because of the file name structure. The original code's default file name should end with a suffix, e.g. `.jp2` in `Dog 20X Caspase-3 (1).jp2`. However, in some datasets, the file name will not have the suffix, for example, dataset 178.